src/profile/helpers/__tests__/editData.test.ts

```diff
@@ -254,6 +288,234 @@ describe('editData.ts ', () => {
       });
     });
   });
+  multiItemDataTypes.forEach(dataType => {
+    describe(`addItem `, () => {
+      it(`creates new ${dataType} item with correct properties`, () => {
+        const { newItem } = createTestDataWithNewItem(dataType);
+        expect(newItem.id).toEqual('');
+        expect(newItem.primary).toBeFalsy();
+        expect(newItem.saving).toBeUndefined();
+        if (dataType === 'emails') {
+          expect((newItem.value as EmailValue).email).toBeDefined();
+        } else if (dataType === 'phones') {
+          expect((newItem.value as PhoneValue).phone).toBeDefined();
+        } else if (dataType === 'addresses') {
+          expect((newItem.value as AddressValue).address).toBeDefined();
+        }
+      });
+      it(`updateItemAndCreateSaveData updates the new ${dataType} item and it is in returned formValues`, () => {
```

Comment:
I wonder if the language of this sentence could be improved? Im not sure I got the ending part of it?

Reply:
Fixed.